docs/src/pages/demos/bottom-navigation/SimpleBottomNavigation.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+import RestoreIcon from '@material-ui/icons/Restore';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+
+const useStyles = makeStyles((_: Theme) =>
```

Comment:
You don't need the theme in this example. @eps1lon I'm wondering do we need `createStyles()`, can it be as simple as?

```tsx
const useStyles = {
  root: {
    width: 500,
  },
};
```

Reply:
you're right, it can be simplified. pushing the update.